packages/SketchMorph2-Core.package/M2Backend.class/instance/formChanged.st

```diff
@@ -1,5 +1,5 @@
 accessing
 formChanged
-	self pen: self defaultPen.
+	self reloadCurrentPen.
 	self eraser: self defaultEraser.
-	self isErasing: false.
\ No newline at end of file
+	self isErasing: false
```

Comment:
Is `self isErasing: false` even necessary anymore since we call `self perform: self currentTool` anyway?

Reply:
In fact, this leads to the eraser behaving like the pencil after resizing.